src/ui/server/api/keystone/application/buyer.ts

```diff
@@ -0,0 +1,40 @@
+import apollo from '../../../graphql/apollo';
+import updateBuyerMutation from '../../../graphql/mutations/update-application/buyer';
+import { ApolloResponse } from '../../../../types';
+
+const buyer = {
+  update: async (id: string, update: object) => {
+    try {
+      console.info('Updating application buyer country');
+
+      const variables = {
+        where: {
+          id,
+        },
+        data: update,
+      };
+
+      const response = (await apollo('POST', updateBuyerMutation, variables)) as ApolloResponse;
+
+      if (response.errors) {
+        console.error('GraphQL error updating application buyer ', response.errors);
+      }
+
+      if (response?.networkError?.result?.errors) {
+        console.error('GraphQL network error updating application buyer ', response.networkError.result.errors);
+      }
+
+      if (response?.data?.updateBuyer) {
+        return response.data.updateBuyer;
+      }
+
+      console.error(response);
+      throw new Error('Updating application buyer country');
+    } catch (err) {
+      console.error(err);
+      throw new Error('Updating application buyer country');
```

Comment:
Can we change the logs to `Updating application buyer` since it could not be country specific

Reply:
this is only being used for updating the country

Reply:
Yes currently, but the actual GQL call is accepting an object and the object then updates the whole buyer

Reply:
changed